lms/static/scripts/frontend_apps/services/test/grading-test.js

```diff
@@ -45,20 +45,20 @@ describe('GradingService', () => {
         student: {
           LISResultSourcedId: 0,
           LISOutcomeServiceUrl: 'url',
+          lmsId: 'studentId',
         },
         grade: 1,
       });
-      assert.isTrue(
-        fakeAPICall.calledWithMatch({
-          authToken: 'dummy-token',
-          path: '/api/lti/result',
-          data: {
-            lis_result_sourcedid: 0,
-            lis_outcome_service_url: 'url',
-            score: 1,
-          },
-        })
-      );
+      assert.calledWithMatch(fakeAPICall, {
```

Comment:
I changed this to this style I've seen in other tests, the error message is much better.

I don't know what I'm doing here so there might be a trade off I'm not aware off. Happy to return to the `isTrue` style.

Reply:
👍 - Using the more specific asserts which generate better errors is good.